product_security/playbooks/security_vulnerability_playbook.json

```diff
@@ -0,0 +1,142 @@
+{
+    "categorize_channel_enabled": true,
+    "category_name": "Incidents",
+    "channel_name_template": "JIRA_ID: TITLE",
+    "checklists": [
+        {
+            "items": [
+                {
+                    "command": "/header **SEVERITY: severity** | [Playbook](playbook_link) | [Run](run_link) | [JIRA Ticket](jira_link) | [Hackerone Report](h1_report_link)",
+                    "description": "Include the following information:\n- Full Title of the Report\n- Link to the JIRA Report (if available)\n- Link to the HackerOne Report (if available)\n- Severity of the Ticket\n",
+                    "title": "Set Channel Header/Purpose"
+                },
+                {
+                    "description": "Invite SecOps Team if the issue is related to Cloud/Infrastructure.\nInvite Management Team according to the severity of the ticket. \n\n```\nYou have been invited you to this channel to support the remediation of a security vulnerability observed in \u003cplaceholder\u003e. Details of the vulnerability are available in the [JIRA Ticket](https://mattermost.atlassian.net/browse/\u003cticket_id\u003e). \n\nPlease let us know if you need any additional information and additionally to invite other team members on a per-need basis.\n\nThank you for your assistance.\n```",
```

Comment:
My only recommendation/change to this (which we will be implementing internally too) is regarding the `Invite SecOps Team`. In the future we will want to update this where if it's a significant vulnerability we would have the playbook page the SecOps on-call along with tagging+inviting them, similar to the cloud-oncall it's doing today.  For lower severity investigation we would just tag and invite the on-call.  This is to prevent randomizing the entire SecOps team and targeting the one person responsible for possible incidents during that window.

Reply:
Hi @iyampaul 
I have updated the Playbook accordingly. I have now added this under the Investigation stage. Hope this addresses your suggestion.

Reply:
Yep, thanks @srkgupta ! The two aliases don't exist yet today but we'll get them setup shortly to meet it :)